src/ray/raylet/scheduling/cluster_resource_manager.cc

```diff
@@ -82,6 +82,24 @@ bool ClusterResourceManager::UpdateNode(scheduling::NodeID node_id,
     local_view.object_pulls_queued = resource_data.object_pulls_queued();
   }
 
+  if (resource_data.resources_normal_task_changed() &&
```

Comment:
this function need unit test too

Reply:
After thinking about it, the resources of normal task are relatively independent. I decided to put this logic in a separate method temporarily to prevent affecting the existing logic.
The test is added.